modules/Setup/WiFiSetup.cpp

```diff
@@ -379,4 +427,151 @@ bool WpaCliSetup::is_wifi_interface(const std::string& interface) {
     return std::filesystem::exists(path);
 }
 
+int Ssid::standard(std::uint8_t c) {
+    int output{-1};
+    if (c == '\\') {
+        handler = &Ssid::backslash;
+    } else {
+        output = static_cast<int>(c);
+    }
+    return output;
+}
+
+int Ssid::backslash(std::uint8_t c) {
+    int output{static_cast<int>(c)};
+    handler = &Ssid::standard;
+    switch (c) {
+    case '"':
```

Comment:
use `[[fallthrough]];` if that's intended

Reply:
That doesn't appear to be a standard approach.
e.g. https://github.com/EVerest/everest-core/blob/e4b30fac7defcce81b3de9b944de7b9be535a77d/modules/EvseManager/Charger.cpp#L775
Doesn't do this:
```
    switch (cp_event) {
    case CPEvent::CarPluggedIn:
        [[fallthrough]];
    case CPEvent::CarRequestedPower:
        [[fallthrough]];
    case CPEvent::CarRequestedStopPower:
        [[fallthrough]];
    case CPEvent::CarUnplugged:
        [[fallthrough]];
    case CPEvent::BCDtoEF:
        [[fallthrough]];
```
The example in 
https://en.cppreference.com/w/cpp/language/attributes/fallthrough
uses [[fallthrough]]; between case 2 and 3 but not between case 1 and 2.

So I don't think adding [[fallthrough]]; is needed here: 
```
    case '"':
    case '\\':
        break;
```

Reply:
You're right :+1: